app/helpers/application_helper.rb

```diff
@@ -16,6 +16,9 @@ def i18n_error_count(count)
   end
 
   def format_content(content)
-    safe_join(content.split("\n"), tag.br)
+    uri_reg = URI::DEFAULT_PARSER.make_regexp(%w[http https])
```

Comment:
```suggestion
    uri_regexp = URI::DEFAULT_PARSER.make_regexp(%w[http https])
```

正規表現はregexまたはregexpと略すことが多いですね。
右辺のメソッドもregexpになってるのでそれに合わせましょう。

Reply:
ありがとうございます！
修正しました～